pdal/Dimension.json

```diff
@@ -385,6 +385,18 @@
     "description": "Distance metric related to a point's nearest neighbors."
     },
     {
+    "name": "TextureX",
+    "alt_names": ["tx", "texture_x", "texturex" ],
```

Comment:
Just curious if these alternate names came from real OBJ files.

Reply:
yes I saw them in outputs.